src/content/add-to-app/ios/project-setup.md

```diff
@@ -393,99 +400,105 @@ as described in Option B.
 
 ## Local Network Privacy Permissions
 
-On iOS 14 and higher, enable the Dart multicast DNS
-service in the Debug version of your app
-to add [debugging functionalities such as hot-reload and
-DevTools][] using `flutter attach`.
+On iOS 14 and later, enable the Dart multicast DNS service in the
+Debug version of your app to add
+[debugging functionalities such as hot-reload and DevTools][]
+using `flutter attach`.
 
 :::warning
 This service must not be enabled in the **Release**
 version of your app, or you might experience App Store rejections.
 :::
 
-One way to do this is to maintain a separate copy of your app's Info.plist per
-build configuration. The following instructions assume
-the default **Debug** and **Release**.
+Maintain a separate copy of your app's Info.plist per build configuration.
+The following instructions assume the default **Debug** and **Release**.
 Adjust the names as needed depending on your app's build configurations.
 
-<ol>
-<li>
+1. Rename your app's **Info.plist** to **Info-Debug.plist**.
 
-Rename your app's **Info.plist** to **Info-Debug.plist**.
-Make a copy of it called **Info-Release.plist** and add it to your Xcode project.
+1. Copy the plist and call it **Info-Release.plist**.
 
-{% render docs/app-figure.md, image:"development/add-to-app/ios/project-setup/info-plists.png", alt:"Info-Debug.plist and Info-Release.plist in Xcode" %}
+1. Add **Info-Release.plist** to your Xcode project.
 
-</li>
+    {% render docs/captioned-image.liquid,
+    image:"development/add-to-app/ios/project-setup/info-plists.png",
+    caption:"Info-Debug.plist and Info-Release.plist in Xcode" %}
 
-<li>
+1. In **Info-Debug.plist** _only_ add the key `NSBonjourServices`.
 
-In **Info-Debug.plist** _only_ add the key `NSBonjourServices`
-and set the value to an array with the string `_dartVmService._tcp`.
-Note Xcode will display this as "Bonjour services".
+1. Set the value to an array with the string `_dartVmService._tcp`.
```

Comment:
Can we make `_dartVmService._tcp` a separate code block so that there's a copy button? Selecting this text to copy it is a little annoying.